src/main/java/org/justinbaur/bankteller/domain/Profile.java

```diff
@@ -0,0 +1,84 @@
+package org.justinbaur.bankteller.domain;
+
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "accounts")
```

Comment:
I'm actually a little confused why you have both the profile and customer info objects being stored in the same collection.  A collection should hold similar objects.

Reply:
Disregard, I was confused because the `CustomerInfo.java` class had the annotation which is not needed since this is your wrapping POJO